scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/ProcessRun.java

```diff
@@ -19,6 +19,7 @@
  * Attempts to process run directories, provided on the command line, through a particular processor
  * and display the results. This is for debugging purposes.
  */
+@SuppressWarnings("all")
```

Comment:
What are all the warnings getting suppressed here? I'm not comfortable if there are suppressed type warnings.

Reply:
Please never suppress all warnings. Specific warnings only, and preferably with a comment explaining why it's safe to do so

Reply:
updated